src/components/JsonEditorContainer/JsonEditorContainer.tsx

```diff
@@ -44,6 +46,19 @@ const getExistingKeys = (json: any, path: (number | string)[]) => {
     });
 }
 
+const getValidInsertItems = (parentPath: (string | number)[]) => {
+    const validationResult = addNode([...parentPath]).resultNode;
+    if (validationResult instanceof ObjectNode) {
+        return validationResult.data;
+    }
+    else if (validationResult instanceof ArrayNode){
```

Comment:
If ArrayNode then return sampleData else return data for all the cases

Reply:
K I'll change that